src/cares_wrap.cc

```diff
@@ -525,6 +544,573 @@ class QueryWrap : public AsyncWrap {
   }
 };
 
+Local<Array> AddrTTLToArray(Environment* env,
+                            ares_addrttl* addrttls,
+                            int naddrttls) {
```

Comment:
`naddrttls` should be `int` because `ares_parse_a_reply` needs `naddrttls` be an integer but not an unsigned integer.